highlight.io/components/QuickstartContent/backend/java/other.tsx

```diff
@@ -0,0 +1,33 @@
+import { siteUrl } from '../../../../utils/urls'
+import { QuickStartContent } from '../../QuickstartContent'
+import {
+	customError,
+	installSdk,
+	initializeSdk,
+	setUpLogging,
+	verifyErrors,
+	sessionUsage,
+} from './shared-snippets'
+
+export const JavaOtherContent: QuickStartContent = {
+	title: 'Java',
+	subtitle: 'Learn how to set up highlight.io on your Java backend.',
+	logoUrl: siteUrl('/images/quickstart/java.svg'),
+	entries: [
+		installSdk,
+		initializeSdk,
+		{
+			title: 'Add Highlight logger.',
+			content:
+				'errors will automatically record raised exceptions and send them to Highlight.',
+			code: {
+				text: `Coming soon`,
+				language: 'java',
+			},
```

Comment:
is this for auto-error-capture for something like spring? perhaps we should remove this for now?

Reply:
Yeah, it's for auto-error-capture, and its currently not supported, so we can remove it for now.

Reply:
thanks! otherwise it's a bit confusing, especially with the word `logger`